databricks/koalas/frame.py

```diff
@@ -6175,6 +6175,102 @@ def count(self, axis=None):
             Frame._count_expr, name="count", axis=axis, numeric_only=False
         )
 
+    def droplevel(self, level, axis=0) -> "DataFrame":
+        """
+        Return DataFrame with requested index / column level(s) removed.
+
+        Parameters
+        ----------
+        level: int, str, or list-like
+            If a string is given, must be the name of a level If list-like, elements must
+            be names or positional indexes of levels.
+
+        axis: {0 or ‘index’, 1 or ‘columns’}, default 0
+
+        Returns
+        -------
+        DataFrame with requested index / column level(s) removed.
+
+        Examples
+        --------
+        >>> df = ks.DataFrame([
+        ...    [1, 2, 3, 4],
+        ...    [5, 6, 7, 8],
+        ...    [9, 10, 11, 12]
+        ... ]).set_index([0, 1]).rename_axis(['a', 'b'])
```

Comment:
Seems like we have a different behaviour from pandas here.

When create a DataFrame using list of list, the type of default columns is going to be a `object` unlike pandas (numeric).

```python
>>> kdf = ks.DataFrame([[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]])
>>> pdf = pd.DataFrame([[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]])
>>> kdf.columns
Index(['0', '1', '2', '3'], dtype='object')  # the type of default columns for Koalas is `object`
>>> pdf.columns
RangeIndex(start=0, stop=4, step=1)  # whereas for pandas is `int64`
>>> pdf.columns.dtype
dtype('int64')
```

cc @ueshin @HyukjinKwon . Should we fix this? or Is there a special reason why Koalas is using such behaviour ?

Reply:
So far column names accept only string or tuple of string.

Reply:
@jijosg , Could you fix this test like the below for now because Koalas has not `rename_axis` yet ?

```python
>>> df = ks.DataFrame(
...     [[3, 4], [7, 8], [11, 12]],
...     index=pd.MultiIndex.from_tuples([(1, 2), (5, 6), (9, 10)], names=["a", "b"]),
... )

>>> df.columns = pd.MultiIndex.from_tuples([
...   ('c', 'e'), ('d', 'f')
... ], names=['level_1', 'level_2'])
```

Reply:
@ueshin Thanks!

Reply:
sure i will make the changes , thanks for looking into this